server/src/main/java/org/apache/iotdb/db/qp/executor/PlanExecutor.java

```diff
@@ -1055,13 +1058,19 @@ private boolean createTimeSeries(CreateTimeSeriesPlan createTimeSeriesPlan)
   private boolean createMultiTimeSeries(CreateMultiTimeSeriesPlan createMultiTimeSeriesPlan) {
     Map<Integer, Exception> results = new HashMap<>(createMultiTimeSeriesPlan.getPaths().size());
     for (int i = 0; i < createMultiTimeSeriesPlan.getPaths().size(); i++) {
-      CreateTimeSeriesPlan plan = new CreateTimeSeriesPlan(createMultiTimeSeriesPlan.getPaths().get(i),
-        createMultiTimeSeriesPlan.getDataTypes().get(i), createMultiTimeSeriesPlan.getEncodings().get(i),
-        createMultiTimeSeriesPlan.getCompressors().get(i),
-        createMultiTimeSeriesPlan.getProps() == null ? null : createMultiTimeSeriesPlan.getProps().get(i),
-        createMultiTimeSeriesPlan.getTags() == null ? null : createMultiTimeSeriesPlan.getTags().get(i),
-        createMultiTimeSeriesPlan.getAttributes() == null ? null : createMultiTimeSeriesPlan.getAttributes().get(i),
-        createMultiTimeSeriesPlan.getAlias() == null ? null : createMultiTimeSeriesPlan.getAlias().get(i));
+      CreateTimeSeriesPlan plan = new CreateTimeSeriesPlan(
+          createMultiTimeSeriesPlan.getPaths().get(i),
+          createMultiTimeSeriesPlan.getDataTypes().get(i),
+          createMultiTimeSeriesPlan.getEncodings().get(i),
+          createMultiTimeSeriesPlan.getCompressors().get(i),
+          createMultiTimeSeriesPlan.getProps() == null ? null
+              : createMultiTimeSeriesPlan.getProps().get(i),
+          createMultiTimeSeriesPlan.getTags() == null ? null
+              : createMultiTimeSeriesPlan.getTags().get(i),
+          createMultiTimeSeriesPlan.getAttributes() == null ? null
+              : createMultiTimeSeriesPlan.getAttributes().get(i),
+          createMultiTimeSeriesPlan.getAlias() == null ? null
+              : createMultiTimeSeriesPlan.getAlias().get(i));
```

Comment:
Recover the format...

Reply:
Actually I have imported `java-google-style.xml` in the root dir before formating this file.Maybe the old format is not suitable in `java-google-style.xml`. As you can see, the default hard warp is 100.
![image](https://user-images.githubusercontent.com/32640567/97674575-b6f19880-1ac8-11eb-81b0-ffa5e2b3ea0e.png)
If you put them on one line, the code on the right will exceed the hard limit.
![image](https://user-images.githubusercontent.com/32640567/97674326-4fd3e400-1ac8-11eb-832d-d59a0ede016c.png)

Reply:
You can rename the `createMultiTimeSeriesPlan` to a shorter one. It's soooo ugly now.

Reply:
Good suggestion!